### PR TITLE
RTS audio recommendation feature

### PR DESCRIPTION
**Pull request by pyby**
When asking for a media recommendation list for an RTS audio, next episodes or clips are provided.
Then, old episodes or clips and recommended.

More details in `docs/RECOMMENDATION.md`. Based on discussions afters @svenduvoisin tests.

@SebastienChauvin Please add in this file the first Play Android version supporting `Playfff` recommendations.

This pull request also fixes the recommended page size for peach requests (49, not 50).

### CODE REVIEW

File: src/main/java/com/example/pfff/service/IntegrationLayerRequest.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    private static final Logger logger = LoggerFactory.getLogger(IntegrationLayerRequest.class);
+    public static final int PORT = 80;
+    private final RestTemplate restTemplate;
+
+    public IntegrationLayerRequest(RestTemplateBuilder restTemplateBuilder) {
+        restTemplate = restTemplateBuilder.build();
+    }
+
+    public EpisodeComposition getEpisodeCompositionLatestByShow(String showURN, ZonedDateTime maxPublishedDate, int pageSize, Environment environment) {
+        String path = "/integrationlayer/2.0/episodeComposition/latestByShow/byUrn/" + showURN + ".json";
```
Comment (amtins) [inline src/main/java/com/example/pfff/service/IntegrationLayerRequest.java]: A stringbuilder would be preferable

File: src/main/java/com/example/pfff/service/RecommendationService.java [hunk 2]
```diff
@@ -22,17 +35,88 @@ public RecommendationService() {
 
     public RecommendedList getRecommendedUrns(String purpose, String urn, boolean standalone) {
```
Comment (amtins) [inline src/main/java/com/example/pfff/service/RecommendationService.java]: It seems this method has many reponsabilities

File: src/main/java/com/example/pfff/service/RecommendationService.java [hunk 3]
```diff
@@ -22,20 +35,98 @@ public RecommendationService() {
...
+            }
+            else if (urn.contains(":audio:")) {
+                Media media = integrationLayerRequest.getMedia(urn, Environment.PROD);
+                if (media == null || media.getType() == LIVESTREAM || media.getType() == SCHEDULED_LIVESTREAM || media.getShow() == null) {
+                    return new RecommendedList();
+                }
+
+                EpisodeComposition episodeComposition = integrationLayerRequest.getEpisodeCompositionLatestByShow(media.getShow().getUrn(), null, 100, Environment.PROD);
+                if (episodeComposition == null) {
+                    new RecommendedList();
```
Comment (amtins) [inline src/main/java/com/example/pfff/service/RecommendationService.java]: Bug ?

Comment (pyby) [reply src/main/java/com/example/pfff/service/RecommendationService.java]: Exact. It was a missing `return`.

**Comment by pyby** [comment]
- RTS audio recommendations is now unit tested.
- The double urn issue is fixed (https://github.com/SRGSSR/pfff/pull/17/commits/10f7356e47872a587f2f0539b2359e971e214a1a).

**Pull request merged**